0.4/Dockerfile

```diff
@@ -32,10 +32,13 @@ ENV JANUS_VERSION=${JANUS_VERSION} \
     janusgraph.storage.directory=/var/lib/janusgraph/data \
     gremlinserver.graph=/etc/opt/janusgraph/janusgraph.properties \
     gremlinserver.threadPoolWorker=1 \
-    gremlinserver.gremlinPool=8
+    gremlinserver.gremlinPool=8 \
+    DEBIAN_FRONTEND=noninteractive
 
 RUN groupadd -r janusgraph --gid=999 && \
     useradd -r -g janusgraph --uid=999 janusgraph && \
+    apt-get update -yqq && \
+    apt-get install -yqq krb5-user && \
```

Comment:
Please add the following line below to clean up the apt cache afterwards to reduce image size:
```
rm -rf /var/lib/apt/lists/* && \
```

Reply:
Done.